OmniKit/MessageTransport/Message.swift

```diff
@@ -89,11 +89,12 @@ struct Message {
         return data
     }
     
-    var fault: PodInfoFaultEvent? {
+    var fault: DetailedStatus? {
         if messageBlocks.count > 0 && messageBlocks[0].blockType == .podInfoResponse,
             let infoResponse = messageBlocks[0] as? PodInfoResponse,
-            infoResponse.podInfoResponseSubType == .faultEvents,
-            let fault = infoResponse.podInfo as? PodInfoFaultEvent
+            infoResponse.podInfoResponseSubType == .detailedStatus,
+            let fault = infoResponse.podInfo as? DetailedStatus,
+            fault.currentStatus.faultType != .noFaults
```

Comment:
This additional test is not sufficient because activation time exceeded fault returns a faultType of .noFaults. Need to add additional testing for a fault if fault.podProgressStatus == activationTimeExceeded.